modules/subchannel/src/meshgenerators/SCMDetailedQuadSubChannelMeshGenerator.C

```diff
@@ -33,9 +33,12 @@ SCMDetailedQuadSubChannelMeshGenerator::validParams()
   params.addRequiredParam<unsigned int>("n_cells", "The number of cells in the axial direction");
   params.addRequiredParam<unsigned int>("nx", "Number of channels in the x direction [-]");
   params.addRequiredParam<unsigned int>("ny", "Number of channels in the y direction [-]");
-  params.addRequiredParam<Real>("gap",
-                                "(Its an added distance between a perimetric pin and the duct: "
-                                "Edge Pitch W = pitch/2 - pin_diameter/2 + gap) [m]");
+  params.addRequiredParam<Real>(
+      "gap",
+      "(The side gap, not to be confused with the gap between pins, this refers to the gap "
+      "next to the duct or else the distance between the subchannel centroid to the duct wall."
+      "Edge Pitch W = (pitch/2 - pin_diameter/2 + gap) [m]");
+  params.renameParam("gap", "side_gap", "08/06/2025");
```

Comment:
```suggestion
  params.renameParam("gap", "side_gap", "08/06/2026");
```
needs to be in the future